src/Bicep.Core/Emit/CompileTimeImports/ArmDeclarationToExpressionConverter.cs

```diff
@@ -66,6 +66,8 @@ internal DeclaredTypeExpression CreateDeclaredTypeExpressionFor(string typePoint
     internal DeclaredVariableExpression CreateDeclaredVariableExpressionFor(string originalName)
         => new(sourceSyntax,
             armIdentifierToSymbolNameMapping[new(ArmSymbolType.Variable, originalName)],
+            // TODO do we need to support type here?
```

Comment:
Is this OK?

Reply:
It's fine to leave this as `null`: the type information is only used on exported variables, and we don't currently have a way to re-export an import.

Reply:
Thanks for confirming!